templates/repo/issue/list.tmpl

```diff
@@ -246,7 +246,7 @@
 							</a>
 						{{end}}
 						{{if .Ref}}
-							<a class="ref" href="{{$.RepoLink}}/src/branch/{{.Ref}}">
+							<a class="ref" href="{{$.RepoLink}}/src/branch/{{.Ref | EscapePound}}">
```

Comment:
```suggestion
							<a class="ref" href="{{$.RepoLink}}/src/branch/{{.Ref | PathEscapeSegments}">
```
Think this would be better. Most of the router code uses it and new instances of template code should use instead of EscapePound (old ones weren't changed just out of not touching too much just to change it)

Reply:
Sounds good. I saw ``EscapePound`` used throughout ``templates/repo/branch/list.tmpl`` so I wanted to keep it consistent :)